hieroplatform/src/main/java/org/hiero/sketch/spreadsheet/BucketsDescriptionEqSize.java

```diff
@@ -0,0 +1,79 @@
+package org.hiero.sketch.spreadsheet;
+
+/**
+ * MetaData for one dimensional buckets of equal size
+ */
+public class BucketsDescriptionEqSize implements IBucketsDescription1D {
+
+    private final double minValue;
+    private final double maxValue;
+    private final int numOfBuckets;
+
+
+    public BucketsDescriptionEqSize(final double minValue, final double maxValue, final int numOfBuckets) {
+        this.minValue = minValue;
+        this.maxValue = maxValue;
+        this.numOfBuckets = numOfBuckets;
+    }
+
+    @Override
+    public int indexOf(final double  item) {
+        if (item < this.minValue || item > this.maxValue)
+            return -1;
+        if (item == this.maxValue)
```

Comment:
Can you change this to item >= this.maxValue?
In principle doubles should never be compared for equality...